### ISSUE TITLE
Project Update #4: Week of February 5th

### ISSUE THREAD

**Issue by sjw82** [sjw82/Midrash#15]
This week we met for the first time and worked together on
- Developing a deeper understanding of our archival resource
- Doing a test run on one verse
- Identifying 5-6 verses per person which we will narrow down in the upcoming week
- Started brainstorming XML criteria
- Reached out to a possible mentor in the religious studies department (thanks to our local religious studies major @Nerdologist)

In this upcoming week we intend to
- Begin to narrow down our verses by exploring the viability of the associated Midrash (based on length and depth of analysis; we're aiming for a Goldie Locks medium length, medium depth)
- Start on a defined list of markup criteria
- Begin to lightly search the history of some of the Midrash we know we'll use to see what historical data we'll be able to utilize (we intend to divide them up once we have a more concrete list and do a deeper historical analysis)
- Decide on the narrative we want our site to tell and explore the perspective of our target audience
- Begin to list site features

**Comment by djbpitt** [comment]
@sjw82 Good write-up! Alongside the comprehensive write-up in a single Issue, you may want to create individual Issues, in parallel, for each combination of text and person. For example, where your plans for next week include “Start on a defined list of markup criteria”, you can create a separate Issue for that (alongside this report), assign it to someone, add a Label, and add it to a Project, all from the Issues tab on GitHub. You may want to ask your project mentor to go through the process with your team at your next meeting, so that you can all practice it.

**Comment by k-busko** [comment]
I'm guessing you already know what the structural part of your XML criteria is going to look like (i.e. verse, line, etc), but I am curious what kind of thematic elements and attributes you are going to use throughout the assignment. Will you be focusing more on the similarities the texts have with ones from other religions or more honed in on the stark differences?

**Comment by khuber116** [comment]
I'm interested in knowing exactly what thematic elements you plan on looking at which I see is something you plan on settling on within this next week. I'm also interested to see how you plan to incorporate the historical analysis because I'm sure there's a variety of ways that could go. Are you planning on looking at interpretations over different time periods, the context that led to a certain interpretation? I'm very interested in keeping up with this project, particularly the narrative you try and move forward with it.
